frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/vm-settings-tab-initial-state.ts

```diff
@@ -44,6 +44,8 @@ export const getInitialVmSettings = (data: CommonData): VMSettings => {
     [VMSettingsField.DESCRIPTION]: {},
     [VMSettingsField.USER_TEMPLATE]: {
       isHidden: hiddenByProviderOrTemplate,
+      initialized: isCreateTemplate || !userTemplateName,
```

Comment:
why don't you agree with  `!(isCreateTemplate && userTemplateName)` ? 

this is more clear that it must be in the context of isCreateTemplate

Reply:
`!(isCreateTemplate && userTemplateName)` is wrong.
in the cases of `isCreateTemplate is true` we want the the `initialized` value to true, so the code that initilize userTemplate will be skipped.

Reply:
![aa](https://user-images.githubusercontent.com/3648838/77654539-7bf71780-6f71-11ea-8de6-8e4be5073ab6.png)

not sure, if we are on the same page..

Reply:
look at the last case:
``` js
isCreateTemplate = true
userTemplateName = "somthing"
=> false
```
if should be `true`, because we do not want to run the template initiation if the mode is `template` or `import`

Reply:
you are right, I am sorry, I got confused by the naming

in that case it should be

```tsx
const isVM = !isCreateTemplate && !isProviderImport;
!(isVM && userTemplateName)
```

Reply:
:+1: